packages/cardano-services/test/cli.test.ts

```diff
@@ -296,6 +296,8 @@ describe('CLI', () => {
                   HANDLE_POLICY_IDS,
                   '--handle-provider-server-url',
                   HANDLE_PROVIDER_SERVER_URL,
+                  '--nft-metadata-provider-server-url',
+                  HANDLE_PROVIDER_SERVER_URL,
```

Comment:
There will not be a 'nft metadata provider', only  the existing 'asset provider' that can be started with `--use-typeorm-asset-provider`. Please add a new test that starts `ServicesNames.Asset` with `--use-typeorm-assets`. See `with typeorm -> stakepool provider server` test as an example.
```suggestion
```

Reply:
This was removed in a previous commit: 3c847f1